experiment/build/gcb_build.py

```diff
@@ -30,10 +31,6 @@
 # Maximum time to wait for a GCB config to finish build.
 GCB_BUILD_TIMEOUT = 13 * 60 * 60  # 4 hours.
 
-# TODO(metzman): Make configurable.
-WORKER_POOL_NAME = (
-    'projects/fuzzbench/locations/us-central1/workerPools/buildpool')
```

Comment:
We require all cloud_params to be [in lower cases](https://github.com/google/fuzzbench/blob/da05da0dcb3680deee9d894d1bfa61edbbf4a90c/experiment/run_experiment.py#L100).
Can we expect `work_pool_name` will be in lower cases?
For example, our previous default value contains an upper-case letter.
Now that this default value is replaced by a field in the config file, do we know for sure that that field will not use upper-case letters?

Reply:
Yeah we can't assume theyre all lowercase.